src/GitTabExpansion.ps1

```diff
@@ -101,6 +101,13 @@ function script:gitCommands($filter, $includeAliases) {
     $cmdList | Sort-Object
 }
 
+function script:lastCheckouts($filter){
+    git reflog |
```

Comment:
Two suggestions here:

1. Specify an explicit `--format=%gs` (I think?) to get only the log output you need. There might be other flags you can also specify to optimize the operation (e.g. `--no-decorate`).
2. `reflog` can get quite long under active development, so I would add a `--max-count=<number>` to avoid scanning thousands of references.

Reply:
I came up with two options and I'm not sure which will be better 🤔
1) ` git log --walk-reflogs --max-count=100 --format=%gs`
get more number of reflogs and then search for "'checkout: moving from (.*) to (.*)'" pattern from result.
2) `git log --walk-reflogs --max-count=10 --grep-reflog='checkout: moving from ' --fixed-strings --format=%gs`
get less number of reflogs but already filtered to one with data that we need (containing "checkout: moving from" in reflog subject).

For me option 2) seems more tempting because you get less data from git command. However the performance can depend on how big is you reflog and how many entries it has that are not what we are looking for (we want only reflogs with "checkout: moving from" subject). In other words you don't know how many reflogs git will traverse to get you list of 10 reflogs you want.

I think 2) is fine. How many crazy operations (like rebases of huge number of commits) you would have to do between switching branch 10 times to have huge reflog that can affect performance?